src/elements/Section/components/Section.vue

```diff
@@ -4,10 +4,31 @@
 		:is="htmlTag"
 	>
 		<slot name="start" />
+		<RenderTag
```

Comment:
is the RenderTag component needed? from what I can tell, it is not used properly